pkg/setting/settings_zanzana.go

```diff
@@ -16,6 +16,8 @@ type ZanzanaSettings struct {
 	Addr string
 	// Mode can either be embedded or client
 	Mode ZanzanaMode
+	// ListenHTTP enables OpenFGA http server which allows to use fga cli
+	ListenHTTP bool
```

Comment:
I wonder if we should change this to a full http address `HttpAddr` so you can specify on what address to listen on

Reply:
Yes, I need to make it configurable